src/math.ts

```diff
@@ -0,0 +1,37 @@
+import { generatePercentChanceToFlake, log, PERCENT_CHANCE_OF_SUCCESS } from './utils'
+
+export const clonedMathRandom = Math.random;
+export const clonedMathFloor = Math.floor;
+export const clonedMathCeil = Math.ceil;
+
+Math.random = function () {
+  if (generatePercentChanceToFlake() > PERCENT_CHANCE_OF_SUCCESS) {
+    if (generatePercentChanceToFlake() > PERCENT_CHANCE_OF_SUCCESS) {
+      // Doubly-rare flake: return 3.14159265359
+      log('Math.random flaking (rare!)');
+      return 3.14159265359;
+    }
+    log('Math.random flaking');
+    const unixStr = `${Date.now()}`; // could actually be yesterday via enflake
+    let reversed = "";
+    for (let i = 0; i < unixStr.length; i++) {
+      reversed += unixStr[unixStr.length - 1 - i];
+    }
+    return parseFloat(`0.${reversed}`);
```

Comment:
Let's have the original date.now exported in utils and have that used here instead of just relying on the built-in Date.now
That way, if Date.ts gets some issues then Math.ts won't be affected.

```suggestion
    const timestamp = `${Date.now()}`;
    let reversedTimeStamp = "";
    for (let i = timestamp.length - 1; i >= 0; i--) {
      reversedTimeStamp += timestamp[i];
    }
    return parseFloat(`0.${reversedTimeStamp}`);
```

Reply:
P.S. Good call on using a for loop- I initially squinted my eyes and realized it'd be better to use it since the other built-in methods might have been enflaked.

It's hard to be evil.